rimage/file_format.h

```diff
@@ -18,14 +18,24 @@
 #define SND_SOF_LOGS_SIG_SIZE	4
 #define SND_SOF_LOGS_SIG	"Logs"
 
+#define SND_SOF_UIDS_SIG_SIZE	4
+#define SND_SOF_UIDS_SIG	"Uids"
+
 /*
  * Logs dictionary file header.
  */
 struct snd_sof_logs_header {
 	unsigned char sig[SND_SOF_LOGS_SIG_SIZE]; /* "Logs" */
-	uint32_t base_address;	/* address of log entries section */
-	uint32_t data_length;	/* amount of bytes following this header */
-	uint32_t data_offset;	/* offset to first entry in this file */
+	uint32_t base_address;		/* address of log entries section */
+	uint32_t data_length;		/* amount of bytes following this header */
+	uint32_t data_offset;		/* offset to first entry in this file */
```

Comment:
This extra whitespace can live in a separate cleanup commit?

Reply:
Fixed